api/database/migrations/2024_04_10_200445_combine-assessment-result-note-fields.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        DB::statement("UPDATE assessment_results SET skill_decision_notes = CONCAT(
+            'Decision notes: ', skill_decision_notes, CHR(13),CHR(10),
+            'Notes for this assessment: ', assessment_notes, CHR(13),CHR(10),
+            'Other justification notes:  ', other_justification_notes
+        )");
+
+        Schema::table('assessment_results', function (Blueprint $table) {
+            $table->dropColumn('assessment_notes');
+            $table->dropColumn('other_justification_notes');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('pool_candidates', function (Blueprint $table) {
```

Comment:
Oof, this is a bit tough.  This really doesn't revert the state to the way it was before but it would be tough to parse out the data from skill_decision_notes back to the original columns.  Probably not worth working it out unless we have to.  I guess this feature isn't widely rolled out yet, anyway.